basic_module/account/src/impls.rs

```diff
@@ -36,3 +37,32 @@ impl CheckTxHandler for Handler {
         Ok(())
     }
 }
+
+#[allow(dead_code)]
+pub struct Executor {}
+
+impl TransactionExecutor for Executor {
+    fn execute_transactions(
+        &self,
+        transactions: &[SignedTransaction],
+    ) -> Result<Vec<TransactionExecutionOutcome>, ()> {
+        let mut total_additional_fee: u64 = 0;
+        let mut total_min_fee: u64 = 0;
+
+        for signed_tx in transactions {
+            total_additional_fee += signed_tx.tx.fee;
```

Comment:
maybe the total_additional_fee should be signed_tx.tx.fee - signed_tx.tx.action.min_fee()

Reply:
Oh, I forgot it. I just modified it. Thanks